app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java

```diff
@@ -71,7 +72,7 @@ public DeleteHelper(MediaWikiApi mwApi,
      * @param reason
      * @return
      */
-    private boolean delete(Media media, String reason) {
+    private boolean delete(Context context, Media media, String reason) {
```

Comment:
I think context is not used in this method anymore, so please keep the method as it was previously